core/biogeochem/POPLUC.F90

```diff
@@ -665,9 +687,6 @@ SUBROUTINE POPLUCStep(POPLUC,year)
          if (POPLUC%ptog(g) .gt. 0.0_dp) &
               CALL execute_luc_event('PRIMF','C3ANN',POPLUC%ptog(g),g,POPLUC)
 
-         ! if (POPLUC%stog(g) .gt.0.0_dp) &
-         !      CALL execute_luc_event('SECDF','C3ANN',POPLUC%stog(g),g,POPLUC)
-
          if (POPLUC%gtop(g) .gt.0.0_dp) &
```

Comment:
Not the topic of this particular PR - but should we remove `%gtop` from the code base entirely?

Reply:
Fine with me. I think the logic that nothing can be converted to p is reasonable.